restaurant-front/src/modules/bartender/pages/bartender-dashboard/bartender-dashboard.component.ts

```diff
@@ -47,6 +47,10 @@ export class BartenderDashboardComponent implements OnInit {
   isNewOrders: boolean =false;
   isAcceptedOrders: boolean =false;
   idOfDrink: string ="";
+  isNewOrderItems: boolean =false;
+  isAcceptedOrderItems: boolean =false;
+  idOrder: any;
+  isAdding:boolean =false;
```

Comment:
Maybe have a single field that is a string and use it in the ngIf to render a component. Also you can remove the methods that only change one state, instead just put (click)="this.state='1'". Think about this, it will reduce a lot of lines of code.